faqs/galaxy/workflows_variables.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Rename outputs in a workflow using workflow variables
+area: workflows
+box_type: tip
+layout: faq
+contributors: [delphine-l]
+optional_parameters:
+  var_name: Name of the workflow Variable
+examples:
+  Use a Species Name:
+    var_name: "Species Name"
+---
+
+Workflow text inputs can be used as parameters for tools but also in the workflow editor for renaming files. 
+
+{% if include.var_name %}
```

Comment:
instead of this big if case, you can also just do `{{ include.var_name | default: "Species Name" }}`  to default to a value if no include variable provided